cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -543,8 +614,9 @@ func prepareDestroyMocks(mockContext *mocks.MockContext) {
 
 	// Purge App configuration
 	mockContext.HttpClient.When(func(request *http.Request) bool {
-		return request.Method == http.MethodPost && strings.Contains(request.URL.Path,
-			"deletedConfigurationStores/ac-123/purge")
+		return request.Method == http.MethodPost && (strings.Contains(request.URL.Path,
```

Comment:
NIT: Could you see if it is possible to insert the newlines between the boolean operators and not in the `strings.Contains` calls itself?

For example, it might read a little cleaner if we can structure it like this:

```go
return request.Method == http.MethodPost &&
   (strings.Contains(request.URL.Path, "deletedConfigurationStores/ac-123/purge") ||
    strings.Contains(request.URL.Path, "deletedConfigurationStores/ac2-123/purge"))
```